codegen/smithy-dafny-codegen-cli/src/main/java/software/amazon/polymorph/CodegenCli.java

```diff
@@ -171,6 +184,14 @@ private static Options getCliOptions() {
           .required()
           .build()
       )
+      .addOption(
+        Option
+          .builder("ln")
+          .longOpt("library-name")
+          .desc("if generating for a language that uses library names (go, python), the name of the library")
```

Comment:
in this case `uses` ==> `dependency`?

Reply:
I'm not following...
Maybe some examples will answer your question?
```
AwsCryptographyPrimitives: python-module-name --> library-name = aws_crytography_primitives
AwsCryptographicMaterialProviders: python-module-name --> library-name = aws_crytographic_materialproviders
```

Reply:
What about:

`if generating for a language that uses library names (go, python), the name of the library in that language`